### DISCUSSION TITLE
color schemes

### DISCUSSION THREAD

**Discussion by sen1van** [lichess-org/mobile#69] [Ideas]
I think you can add color schemes (at least for android, on ios idk how to do this), it looks amazing to me
<img src="https://user-images.githubusercontent.com/80615886/222919503-389f7db9-ad5f-4479-87e6-28ac1e6c0825.jpg" width="200"><img src="https://user-images.githubusercontent.com/80615886/222919504-72a9d4e9-2596-4ffd-b471-5877ce5704bd.jpg" width="200"><img src="https://user-images.githubusercontent.com/80615886/222919506-67b54170-4821-4632-8198-c12c75caa313.jpg" width="200">

(sorry, how to change image size?)

**Comment by veloce** [comment]
If you have some code to create the color schemes, you can share it in a PR. What could be interesting also is a color scheme that would match the main color of the selected board.
Also how does it look like on the light theme?

**Comment by sen1van** [comment]
<img src="https://user-images.githubusercontent.com/80615886/222969700-2c396b62-5d8d-4b96-8542-5478af0210b9.png" width="300"><img src="https://user-images.githubusercontent.com/80615886/222969692-52816249-e375-4e68-b442-74e8bb2fb1ea.png" width="300">
actually to represent this idea, i just changed the colorSchemeSeed in app.dart, i tried to add a button in settings, but i don't have enough skills.

**Reply by veloce** [reply]
I see. I like the idea because the blue (which is lichess primary colour) is a little boring, especially in the light mode.
